docs/user-guide/node-selection/index.md

```diff
@@ -77,16 +77,16 @@ with a standard set of labels. As of Kubernetes v1.4 these labels are
 * `beta.kubernetes.io/os`
 * `beta.kubernetes.io/arch`
 
-## Alpha features: affinity and anti-affinity
+## Beta features: affinity and anti-affinity
```

Comment:
I would change the title to just "Affinity and anti-affinity". It's fine to mention Beta in the other parts of the doc, but for the title I think simpler is probably better (the Alpha was there as a kind of warning, but now that it's Beta I don't think we need to say anything in the title).